storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2703,36 +2703,47 @@ static bool xtrabackup_read_info(char *filename) {
   FILE *fp;
   bool r = true;
   char mysql_server_version_str[30] = ""; /* 8.0.20.debug| 8.0.20 */
-  unsigned long mysql_server_version;
+  char xtrabackup_version_str[30] = "";   /* 8.0.20.debug| 8.0.20 */
+  unsigned long xb_server_version;
 
   fp = fopen(filename, "r");
   if (!fp) {
     xb::error() << "cannot open " << filename;
     return (false);
   }
-  /* skip uuid, name, tool_name, tool_command, tool_version, ibbackup_version */
-  for (int i = 0; i < 6; i++) {
+  /* skip uuid, name, tool_name, tool_command, tool_version */
+  for (int i = 0; i < 5; i++) {
     char c;
     do {
       c = fgetc(fp);
     } while (c != '\n');
   }
 
+  if (fscanf(fp, "ibbackup_version = %29s\n", xtrabackup_version_str) != 1) {
```

Comment:
This part is confusing.
We have two versions that we care about:
1) Server that we took the backup from
2) PXB that we too the backup

Those two are still required. We should not get rid of server version here.

Reply:
there was a global variable `mysql_server_version` so just rename the local variable `mysql_server_version` to `xb_server_version`.

Reply:
ok, got it